tests/test_compare.py

```diff
@@ -13,13 +14,28 @@
 
 @pytest.fixture(name="config", scope="module")
 def _config() -> Config:
-    """ Fixture that returns a static Config object. """
+    """Fixture that returns a static Config object."""
     return Config(float_absolute_tolerance=ABS_TOL, float_relative_tolerance=REL_TOL)
 
 
 # ===== Unit Tests =========================================
 
 
+@pytest.mark.parametrize(
+    "value, expected, is_equal",
+    [(Unit("meter"), Unit("meter"), True), (Unit("meter"), Unit("feet"), False)],
```

Comment:
Do we need test cases for the situation you mentioned in compare.py where e.g. `expected` could be 'meter' instead of Unit('meter')

Reply:
added 

[c73b03a](https://github.com/MORSECorp/snappiershot/pull/96/commits/c73b03ad9710f2a4092436bfa637e470499bccfb)